src/main/java/uk/gov/companieshouse/pscfiling/api/model/entity/PscIndividualFiling.java

```diff
@@ -11,110 +13,67 @@
 import java.util.function.Consumer;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
-import org.springframework.data.mongodb.core.mapping.Unwrapped;
 
 @Document(collection = "psc_submissions")
 @JsonInclude(JsonInclude.Include.NON_NULL)
-public class PscIndividualFiling implements PscCommunal {
+public class PscIndividualFiling extends PscCommon implements PscCommunal, Touchable {
+    @Override
+    public boolean equals(final Object o) {
+        if (this == o) {
+            return true;
+        }
+        if (o == null || getClass() != o.getClass()) {
+            return false;
+        }
+        if (!super.equals(o)) {
+            return false;
+        }
+        final PscIndividualFiling that = (PscIndividualFiling) o;
+        return Objects.equals(getId(), that.getId())
+                && Objects.equals(getCountryOfResidence(), that.getCountryOfResidence())
+                && Objects.equals(getDateOfBirth(), that.getDateOfBirth())
+                && Objects.equals(getNameElements(), that.getNameElements())
+                && Objects.equals(getNationality(), that.getNationality())
+                && Objects.equals(getResidentialAddress(), that.getResidentialAddress())
+                && Objects.equals(getResidentialAddressSameAsCorrespondenceAddress(),
+                that.getResidentialAddressSameAsCorrespondenceAddress())
+                && Objects.equals(getStatementActionDate(), that.getStatementActionDate())
+                && Objects.equals(getStatementType(), that.getStatementType());
+    }
+
+    @Override
+    public int hashCode() {
+        return Objects.hash(super.hashCode(), getId(), getCountryOfResidence(), getDateOfBirth(),
+                getNameElements(), getNationality(), getResidentialAddress(),
+                getResidentialAddressSameAsCorrespondenceAddress(), getStatementActionDate(),
+                getStatementType());
+    }
 
```

Comment:
Please move these to after the getter methods to be more consistent with the other classes